core/utils/poly-look/src/visualisations/charts/bubble-charts/iconCluster/iconCluster.js

```diff
@@ -0,0 +1,84 @@
+import { BubbleCluster } from "../..";
+
+const bubblePadding = 15;
+
+/**
+ * Visualizes data as a cluster of icon (unicode-emoji or svg) where the value of the icon is represented as the size.
```

Comment:
Does this one actually work with an emoji? With the `html()` call below, it looks to me like only SVG objects work.

Reply:
This is actually true, I got rid of the font code. Can be reintroduced when necessary though. Will update the doc